src/Yasumi/Provider/Canada.php

```diff
@@ -253,4 +258,18 @@ private function calculateLabourDay(): void
             $this->locale
         ));
     }
+
+    private function calculateNationalDayForTruthAndReconciliation()
```

Comment:
Do you have some sources that would provide a description of this holiday?
I looked at this page https://www.statutoryholidays.com/september30-holiday.php and according to that it is **not** a public holiday.

Reply:
I do not know what that page means by "public holiday" in a Canadian context.

If it means days on which every employee in Canada is statutorily entitled to a day off with pay, then there are very few of those.

In Canada, constitutional authority to regulate employment is divided between the federal and provincial/territorial governments. Federally regulated workplaces observe federal statutory holidays. All other workplaces observe the statutory holidays of the province or territory in which they are located. 
For a holiday to be one on which every employee in Canada is entitled to a day off with pay, it would have to be a statutory holiday federally and in every province/territory. It appears that is only true for New Year's Day, Canada Day, Labour Day and Christmas Day.

The page to which you refer correctly notes that National Day for Truth and Reconciliation is a federal statutory holiday. It is not a statutory holiday in any province/territory. So it is not a statutorily required day off with pay for every employee in Canada.

Good Friday, Victoria Day, Thanksgiving Day, Remembrance Day and Boxing Day are all federal statutory holidays.  They are each statutory holidays in some but not all provinces/territories. Hence, they are not statutorily required days off with pay for every employee in Canada.

Reply:
Thank you for the extensive information. Public holidays being non-working days or not vary greatly between countries, and as such cause sometimes confusion. For this library I tend to only consider if a holiday is officially recognized by a local government. Irrespective if that means it is a non-working day or not. Because that is often arbitrary and not always regulated by a government.

This is one of the reasons why Yasumi is not perfect in this regards :) I am still looking into that how to do that better in a future release.

Reply:
So do you consider this a holiday by your definition?

Reply:
Yes, for this library any day can be included. Just a matter of the correct classification.